File: 프로그래머스/lv1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.java
import java.util.*;

class Solution {
    public int solution(int left, int right) {
        int ans = 0;
        
        for (int i = left; i <= right; i++) {
            ans += i % Math.sqrt(i) == 0 ? -i : i;
        }
        
        return ans;
    }
}

File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.kt
import kotlin.math.sqrt

class Solution {
    fun solution(left: Int, right: Int): Int {
        var answer: Int = 0
        for (i in left..right) 
            answer += if (i % sqrt(i.toDouble()) == 0.0) -i else i
        return answer
    }
}